crates/ruma-events/Cargo.toml

```diff
@@ -25,6 +25,7 @@ unstable-pre-spec = ["ruma-identifiers/unstable-pre-spec"]
 
 [dependencies]
 criterion = { version = "0.3.3", optional = true }
+glob = "0.3.0"
```

Comment:
The glob crate will treat a lot more things specially than just `*` and `?` as the Matrix spec says. In ruma-common we use the wildmatch crate which seems to do just these two. Please do the same here.